File: ipman.shared/Entity/FaceCollection.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace ipman.shared.Entity
{
    public class FaceCollection : EntityBase
    {
        public Guid PersonID { get; set; }
        public Person Person { get; set; }
        public List<FaceImage> FaceImages { get; set; }
        public bool IsActive { get; set; }
        public DateTime CreatedUTC { get; set; }
        public DateTime LastUpdatedUTC { get; set; }
    }
}

File: ipman/ClientApp/entity/FaceCollection.ts
import { EntityBase } from "./EntityBase";
import { FaceImage, Person } from "@entity";

export default interface FaceCollection extends EntityBase{
    PersonID: string;
    Person: Person;
    FaceImages: FaceImage[];
    IsActive: boolean;
    CreatedUTC: Date | string;
    LastUpdatedUTC: Date | string;
}